profiler: Fix silently dropping profiler events (#5902)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

  * Updated Python scripts to use multiple processes that communicate over sockets.
  * Increase the number of supported profiler events.

>   * Added a special profiler event indicating that the profiler's data buffer has overflowed, some events has been dropped and a device will no longer send events.

Reviewer: ```suggestion
  * Added a special profiler event for indicating a situation where the profiler's data buffer has overflowed and some events have been dropped, which causes the device to stop sending events.
```
